frontend/src/components/ui/copy-field.ts

```diff
@@ -56,19 +56,22 @@ export class CopyField extends TailwindElement {
 
   render() {
     return html`
-      <div role="group">
+      <div
+        role="group"
+        class=${clsx(
+          "rounded border",
+          this.filled && "bg-slate-50",
+          this.monostyle && "font-monostyle",
+        )}
+      >
         <label
-          class="${classMap({
-            hidden: !this.label && !this._slottedChildren,
-          })} mb-1.5 inline-block font-sans text-xs leading-[1.4] text-neutral-800"
+          class="${clsx(
+            "mb-1.5 inline-block font-sans text-xs leading-[1.4] text-neutral-800",
+            !this.label && !this._slottedChildren && "hidden",
```

Comment:
Out of scope, but noting since I think I missed this in a previous PR: lit provides [`queryAssignedElements`](https://lit.dev/docs/api/decorators/#queryAssignedElements) that does `get _slottedChildren`.